superset-frontend/src/views/CRUD/data/query/QueryList.tsx

```diff
@@ -0,0 +1,356 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { useMemo } from 'react';
+import { t, styled } from '@superset-ui/core';
+import moment from 'moment';
+
+import withToasts from 'src/messageToasts/enhancers/withToasts';
+import { useListViewResource } from 'src/views/CRUD/hooks';
+import SubMenu, { SubMenuProps } from 'src/components/Menu/SubMenu';
+import { Popover } from 'src/common/components';
+import { commonMenuData } from 'src/views/CRUD/data/common';
+import ListView, { Filters, ListViewProps } from 'src/components/ListView';
+import Icon, { IconName } from 'src/components/Icon';
+import Tooltip from 'src/common/components/Tooltip';
+import SyntaxHighlighter from 'react-syntax-highlighter/dist/cjs/light';
+import sql from 'react-syntax-highlighter/dist/cjs/languages/hljs/sql';
+import github from 'react-syntax-highlighter/dist/cjs/styles/hljs/github';
+
+SyntaxHighlighter.registerLanguage('sql', sql);
+
+const TopAlignedListView = styled(ListView)<ListViewProps<QueryObject>>`
+  table .table-cell {
+    vertical-align: top;
+  }
+`;
+
+const StyledSyntaxHighlighter = styled(SyntaxHighlighter)`
+  height: 105px;
```

Comment:
maybe something like this 
```suggestion
height: ${({ theme }) => theme.gridUnit * 26}px;
```